pkg/cache/cache.go

```diff
@@ -443,13 +446,22 @@ func (c *Cache) UpdateClusterQueue(cq *kueue.ClusterQueue) error {
 func (c *Cache) DeleteClusterQueue(cq *kueue.ClusterQueue) {
 	c.Lock()
 	defer c.Unlock()
+	ctx := context.TODO()
+	logger := ctrl.LoggerFrom(ctx)
 	curCq := c.hm.ClusterQueue(cq.Name)
 	if curCq == nil {
 		return
 	}
-	if features.Enabled(features.LocalQueueMetrics) {
-		for _, q := range c.hm.ClusterQueue(cq.Name).localQueues {
-			metrics.ClearLocalQueueCacheMetrics(metrics.LQRefFromLocalQueueKey(q.key))
+	if features.Enabled(features.LocalQueueMetrics) && metrics.GetLocalQueueMetrics().Enabled {
+		for _, q := range curCq.localQueues {
+			lqSlice := strings.Split(q.key, "/")
+			lq := &kueue.LocalQueue{}
+			err := c.client.Get(ctx, types.NamespacedName{Name: lqSlice[1], Namespace: lqSlice[0]}, lq)
+			if err != nil {
+				logger.Error(err, "Could not get LocalQueue, skipping")
```

Comment:
```suggestion
				logger.Error(err, "skip to clear localQueue metrics")
```

I think the `err` already have similar message as `Could not get LocalQueue`